tests/checks/default/checker_test.sh

```diff
@@ -0,0 +1,8 @@
+bindir=$1
```

Comment:
This doesn't seem connected to anything.

Reply:
Oops, this was a vestige of a previous testing attempt. Removed.

Reply:
Oops, this was a vestige of a previous attempt at testing. Removed.

Reply:
It looks like this is still present in the new location. Maybe the removal didn't get committed / pushed?

Reply:
I have deleted this in the latest commit.